src/main/java/com/notably/logic/commands/HelpCommand.java

```diff
@@ -0,0 +1,21 @@
+package com.notably.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import com.notably.model.Model;
+
+/**
+ * Represent a command that enable/disables the help window.
+ */
+public class HelpCommand extends Command {
+    public static final String COMMAND_WORD = "help";
+
+    @Override
+    public void execute(Model notablyModel) {
+        requireNonNull(notablyModel);
+        if (notablyModel.isHelpOpen()) {
+            notablyModel.setHelpOpen(false);
+        }
```

Comment:
There needs to be an else clause here, if not the last statement will get executed always.

Reply:
Thanks for spotting that! Corrected the mistake.